### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by zacharyweston** [zacharyweston/codelab-advanced-css-selectors#1]
Hey @chrisvfritz, can you take a look at this? It's [hosted here](http://thin-fog.surge.sh) and meets the following [project](http://lansingcodelab.com/lessons/advanced-css-selectors/1) criteria:
- [x] not have any classes or ids included anywhere in the HTML (we can do a lot without them!)
- [x] include a list where the background alternates every other item (hint: use the `:nth-child` selector)
- [x] include a visible change on list items when you hover over them with your mouse
- [x] include special, visible styles on only the first and last elements in the list
- [x] include at least one empty list item that your CSS will hide using the `:empty` selector
- [x] return no errors (warnings are OK) when the HTML document is uploaded to the [W3C Markup Validation Service](http://validator.w3.org/#validate_by_upload)
- [x] return no errors or warnings when contents of the CSS file are pasted into [CSS Lint](http://csslint.net/)

I tried to use CSS selectors to only apply the alternating background to the visible elements, but it appears that I would need to use jQuery or Javascript in order to do that.  Is that true?

**Comment by chrisvfritz** [comment]
That is true! I sneakily engineered that issue in this lesson so that you would experience some of the limitations of CSS first-hand. :stuck_out_tongue_winking_eye: And... this meets all the project criteria and I don't see any significant issues. :shipit: 

I will offer one small note however, regarding your last two selectors:

``` css
ul > li:first-child {
  font-variant: small-caps;
}

ul > li:last-child {
  font-variant: small-caps;
}
```

Since they share the same content, you could actually combine them into a single selector if you wanted, with a comma. It would look like this:

``` css
ul > li:first-child, ul > li:last-child {
  font-variant: small-caps;
}
```

Whether you choose to or not is _somewhat_ up to style though, so it's not a change I'll be enforcing here.
